### PR TITLE
Ryudelta

### PR DESCRIPTION
**Pull request by Grafiters**
# ISSUE NUMBER 101
feature for search square root binary and count factorial number

Screenshot from 2024-10-14 09-59-12
new calculation button

image
seach square root of binary

image
count binary factorial

image
add new styles css for modal in dark-themes

### CODE REVIEW

File: js/binaryCalculator.js [hunk 1]
```diff
@@ -276,6 +276,8 @@ toggleThemeButton.addEventListener("click", () => {
 });
 
 function openModal(message){
+    console.log(`this is modal`);
```
Comment (Alitindrawan24) [inline js/binaryCalculator.js:279]: remove this plz

File: js/binaryCalculator.js [hunk 2]
```diff
@@ -401,4 +403,53 @@ function calculateParityBit() {
...
+        decimalValue = binary;
+    } else {
+        throw new Error("Input must be a binary string or a number.");
+    }
+
+    for (let i = factorial+1; i <= decimalValue; i++) {
+        factorial *= i
+    }
+
+    console.log(decimalValue);
```
Comment (Alitindrawan24) [inline js/binaryCalculator.js:451]: remove this plz

File: js/binaryCalculator.js (hunk 1 above)
Comment (Grafiters) [reply js/binaryCalculator.js]: console.log already removed

File: js/binaryCalculator.js (hunk 2 above)
Comment (Grafiters) [reply js/binaryCalculator.js]: console.log already removed

**Comment by Grafiters** [comment]
all my console.log on project already removed

**Comment by ryudelta** [comment]
OPEN NEW PR

**Comment by Grafiters** [comment]
open new pr

**Comment by Alitindrawan24** [comment]
Your PR contain conflict after I'm merged other PR, plz solve the conflict first @ryudelta

**Pull request closed without merging**